bedshift/bedshift.py

```diff
@@ -228,9 +226,10 @@ def _shift(self, row, mean, stdev):
             # check if the region is shifted out of chromosome length bounds
             return None, None
 
-        return row, {0: chrom, 1: start + theshift, 2: end + theshift, 3: "S"}
+        return row, {0: chrom, 1: start + theshift, 2: end + theshift, 3: 'S'}
+
 
-    def shift_from_file(self, fp, shiftrate, shiftmean, shiftstdev, delimiter="\t"):
+    def shift_from_file(self, fp, shiftrate, shiftmean, shiftstdev, delimiter='\t'):
```

Comment:
Can we not undo the format changes from 9ea3ec58c9d69fae7d8da6e20e1964501bd6d76b?

Reply:
you can also just run `black` on the file to re-create them if that helps @hyunjaecho94 . it's deterministic.